oauth2client/client.py

```diff
@@ -336,95 +337,38 @@ class Flow(object):
 class Storage(object):
     """Base class for all Storage objects.
 
-    Store and retrieve a single credential. This class supports locking
-    such that multiple processes and threads can operate on a single
-    store.
+    Store and retrieve a single credential.
     """
 
-    def acquire_lock(self):
-        """Acquires any lock necessary to access this Storage.
-
-        This lock is not reentrant.
-        """
-        pass
-
-    def release_lock(self):
-        """Release the Storage lock.
-
-        Trying to release a lock that isn't held will result in a
-        RuntimeError.
-        """
-        pass
-
-    def locked_get(self):
+    def get(self):
         """Retrieve credential.
 
-        The Storage lock must be held when this is called.
-
         Returns:
             oauth2client.client.Credentials
         """
         _abstract()
```

Comment:
Absolutely nuke it now. An abstract/interface class should look mostly like
```
import abc

class MyAbstractClass(object):
    """My class doc string.

    More documentation.
    """
    __metaclass__ = abc.ABCMeta

    def MyAbstractMethod(self):
        """My abstract method doc string."""
        raise NotImplementedError()
```
.